commands/issue.go

```diff
@@ -39,7 +39,15 @@ var (
 `,
 	}
 
+	cmdCreateIssue = &Command{
+		Key:   "create",
+		Run:   createIssue,
+		Usage: "issue create [-m <MESSAGE>|-f <FILE>] [-l <LABELS>]",
```

Comment:
Oops, you're right. For subcommands, these fields aren't consulted yet. For now, let's keep them up to date still, at least as some sort of "code" documentation, and we'll figure out how to hook subcommands into the global help system.